### PR TITLE
Cleaned up and expanded shooter mount

### PR DESCRIPTION
**Pull request by renatodellosso**

### CODE REVIEW

File: RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java [hunk 1]
```diff
@@ -1,29 +1,47 @@
...
 	{
+		this(subsystem, () -> speed / 50); // Convert from degrees per second to degrees per 20ms
+	}
 
-		shooterMountRotation = Math.max(0, Math.min(180, shooterMountRotation + shooterMountSpeed));
-		shooterMountSubsystem.setGoalRotation(shooterMountRotation);
-
+	public void execute()
+	{
+		double rotation = Math.max(0,
```
Comment (Tr01ler) [inline RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java:42]: Is something supposed to come after

Comment (Tr01ler) [reply RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java:42]: the comma?

Comment (renatodellosso) [reply RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java:42]: Check the next line

Comment (Tr01ler) [reply RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java:42]: Oh, I didn't realize that was in the parentheses as well

File: RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        addRequirements(subsystem);
+    }
+
+    /**
+     * This constructor is the version that uses a constant target
+     * 
+     * @param position in degrees
+     */
+    public RotateShooterMountToPositionCommand(ShooterMountSubsystem subsystem, double position)
+    {
```
Comment (seankid) [inline RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java:37]: doesn't follow code standards

**Review by Tr01ler — APPROVED** [review]

File: RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java (hunk 2 above)
Comment (renatodellosso) [reply RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java:37]: Explain how

**Review by seankid — APPROVED** [review]
the formatting is no adhering to code standard with brackets but otherwise good

File: RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java (hunk 2 above)
Comment (seankid) [reply RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java:37]: the opening brackets aren't on same line as the declaration

**Comment by renatodellosso** [comment]
> the formatting is no adhering to code standard with brackets but otherwise good

That comes from the autoformatter, so don't worry about it.

**Pull request merged**